apps/demos/Demos/Chat/AIIntagration/jQuery/styles.css

```diff
@@ -0,0 +1,42 @@
+.demo-container {
+  display: flex;
+  justify-content: center;
+  padding: 20px;
+}
+
+.dx-chat-messagelist-empty-image {
+  display: none;
+}
+
+.dx-chat-messagelist-empty-message {
+  font-size: var(--dx-font-size-heading-5);
+}
+
+.dx-chat-messagelist-empty-prompt {
+  width: 270px;
```

Comment:
It strange to set some with for text. Did you consider all size model and all themes?

Reply:
[0f60119da81def3f3f4e8a295eaefda87674892c]